crates/forge/tests/e2e/common/runner.rs

```diff
@@ -1,56 +1,33 @@
 use assert_fs::fixture::{FileWriteStr, PathChild, PathCopy};
 use assert_fs::TempDir;
 use camino::Utf8PathBuf;
-use fs_extra::dir::{copy, CopyOptions};
+// use fs_extra::dir::{copy, CopyOptions};
 use indoc::formatdoc;
 use shared::command::CommandExt;
 use shared::test_utils::node_url::node_rpc_url;
 use snapbox::cmd::{cargo_bin, Command as SnapboxCommand};
-use std::fs::{create_dir_all, remove_dir_all};
+// use std::fs::{create_dir_all, remove_dir_all};
 use std::path::{Path, PathBuf};
 use std::process::Command;
 use std::str::FromStr;
-use std::sync::LazyLock;
+// use std::sync::LazyLock;
 use std::{env, fs};
 use test_utils::tempdir_with_tool_versions;
 use toml_edit::{value, DocumentMut};
 use walkdir::WalkDir;
 
-/// To avoid rebuilding `snforge_std` and associated plugin for each test, we cache it in a directory and copy it to the e2e test temp directory.
-static BASE_CACHE_DIR: LazyLock<PathBuf> =
-    LazyLock::new(|| init_base_cache_dir().expect("Failed to initialize base cache directory"));
-
-fn init_base_cache_dir() -> anyhow::Result<PathBuf> {
-    let cache_dir_path = env::current_dir()?.join(".forge_e2e_cache");
-    if cache_dir_path.exists() {
-        remove_dir_all(&cache_dir_path)?;
-    }
-    create_dir_all(&cache_dir_path)?;
-    let cache_dir_path = cache_dir_path.canonicalize()?;
-
-    let snforge_std = PathBuf::from("../../snforge_std").canonicalize()?;
-
-    SnapboxCommand::new("scarb")
-        .arg("build")
-        .current_dir(snforge_std.as_path())
-        .env("SCARB_CACHE", &cache_dir_path)
-        .assert()
-        .success();
-
-    Ok(cache_dir_path)
-}
-
 pub(crate) fn runner(temp_dir: &TempDir) -> SnapboxCommand {
-    copy(
-        BASE_CACHE_DIR.as_path(),
-        temp_dir.path(),
-        &CopyOptions::new().overwrite(true).content_only(true),
-    )
-    .unwrap();
+    SnapboxCommand::new(snforge_test_bin_path()).current_dir(temp_dir)
+}
 
-    SnapboxCommand::new(cargo_bin!("snforge"))
-        .env("SCARB_CACHE", temp_dir.path())
```

Comment:
Did you check that having shared cache won't break anything?

Reply:
It shouldn't tbh, this is the way it was before so...
It obviously needs to be re-evaluated if we want to add this kind of logic again for nextest (#2443) but ATM it seems obsolete.